atlante/filestore/multi/multi.go

```diff
@@ -0,0 +1,165 @@
+package multi
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/gdey/errors"
```

Comment:
why do we need an external error package?

Reply:
It has better wrap methods so you can get original errors and sentinel errors that are constants instead of var that the code can change.